docs/en/academic/custom-templates.md

```diff
@@ -1,83 +1,134 @@
+With the source and template files prepared, Zettlr will prepopulate the Pandoc command variables. One of these commands tells Pandoc that the revised source file from step 3 is your input file. Another command adds the revised LaTeX template; if a custom template is not provided, Zettlr will use its [default template](https://github.com/Zettlr/Zettlr/blob/master/source/main/assets/export.tex). To maximise compatibility, this template is a simplified version of the [Pandoc default template](https://github.com/jgm/pandoc/blob/master/data/templates/default.latex).
 
-Pandoc's templating engine is pretty powerful, so we can not fully dive into the templating syntax. The [Pandoc documentation](https://pandoc.org/MANUAL.html#templates) has a long section on that, including all supported variables for the built-in templates.
+> Many of the commands in the default Pandoc template require additional LaTeX packages. To not confuse users who only need the basics, Zettlr's default template strives for maximum compatibility, not perfect PDFs.
 
-### Templates versus "Reference Docs"
+<!-- TO CHECK: Regarding the next line, I was not able to find the 'table of contents' option in the project propertie; it seems to be much simpler and only asks about what format to export the project in and also about what files to ignore via a glob pattern (there is also a lot of little icons visible at the botton of this project properties menu that seem to not do anything...).-->
 
-A final preconsideration is the differentiation between templates and reference documents. Most file formats are text-based, so these will use a template (e.g., HTML or LaTeX). A few, however, such as Word and LibreOffice, require so-called "reference documents". A reference document is basically a Word or LibreOffice file that you can open and modify its styles (for example, for headings level 1, or paragraphs). Then, you can save that and tell Pandoc to use the reference document to export your file. Inside these reference documents, you can not make use of metadata variables, unfortunately.
+If you indicated in your project settings that you want a table of contents, the relevant command will be included in the Pandoc command variables. This will cause XeLaTeX to run **twice**: once to build the PDF and determine where the headings will be located, and once to build the PDF with the table of contents included. 
 
-> For more information on reference documents, see the section on [`--reference-doc` in the Pandoc manual](https://pandoc.org/MANUAL.html#options-affecting-specific-writers).
+<!-- TO CHECK: Regarding the next line, the TOC flag and the whole Pandoc command found in the Advanced tab in the Preferences does not seem to exist anymore. Should the line be deleted? Or does this line now refer to PDF  in Preferences -> Asset Manager? If so, the --toc flag is set to false, which does not fit with the line below. -->
```

Comment:
The Pandoc command as well as the PDF preferences are gone. Pandoc is now being customized via the defaults files (in the assets manager), otherwise, see my comment above.

Reply:
Perfect. I will delete this line.